### DISCUSSION TITLE
Welcome to PSG-LookyFeely Discussions!

### DISCUSSION THREAD

**Discussion by definite-d** [definite-d/PSG-LookyFeely#10] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by WarnerMeinshegran** [comment]
Hello, I'm blabla_lab.
I really love your application, especially after the new update (2.0), so I have made a Linux version for themera.
There is a problem for making a theme from image, it crashes or generates a blank window.
Anyway, most of it works. I don't know where to share the file. Im a  beginner at github

**Reply by definite-d** [reply]
Hi @blabla-labALT !

Thank you for your interest in the Themera project, and your efforts to compile it for Linux.

Regarding the crashes and blank windows for generating themes from images, I'd appreciate it if you could run the code (uncompiled) in an IDE and let me know what the error traceback looks like to point me in the right direction. I think opening an issue for these errors is necessary.

In the meantime, I'll perform testing independently to see if I can find the issue. As for sharing the file, you could fork the repository and upload it in a folder on your version.

Once again, thanks for your support. It's really appreciated.

**Reply by WarnerMeinshegran** [reply]
I have tested it again, and it worked completely fine, so I will compile it again and publish it to GitHub, just to make sure that everything is working.
For the errors, there are none except some GTK warnings and errors that didn't affect the application at all, and I don't think you will be able to fix them because they are from Qt library, probably.

**Reply by definite-d** [reply]
Glad to hear that it's working, @blabla-labALT . Thank you for porting the project to Linux; I really appreciate it. I'll make sure I update the ReadMe to point users to your fork if they're need a version for Linux.
